src/database.rs

```diff
@@ -67,7 +67,7 @@ impl Database {
         let execution_plan = execution_engine.plan(Arc::new(physical_plan));
         // println!("{:?}", execution_plan);
         let result = execution_engine.execute(execution_plan);
-        println!("{:?}", result);
+        println!("test: {:?}", result);
```

Comment:
execution result seems better.(I changed it in https://github.com/systemxlabs/tinysql/pull/26)